src/citrine/informatics/executions/generative_design_execution.py

```diff
@@ -0,0 +1,90 @@
+# from deprecation import deprecated
```

Comment:
?

Reply:
Hahaha woops, I meant to delete that. I copied this structure from DesignExecution then updated it to match code found here: https://github.com/CitrineInformatics/platform-backend/blob/generative-design-task/code/orion/ml-config-api/src/main/scala/io/citrine/orion/api/endpoints/GenerativeDesignApi.scala#L69

Reply:
Still need to remove

Reply:
@StevenKauwe this comment stil needs to be addressed